### PR TITLE
[corlib] Replace System and other namespaces with CoreFX sources

### PR DESCRIPTION
**Pull request by EgorBo**
Simple types for now just to see the API-DIFF, more types later.

Part of #8131

### CODE REVIEW

File: mcs/class/corlib/corlib.dll.sources [hunk 1]
```diff
@@ -142,8 +142,8 @@ System/WeakReference_T.cs
 System/WindowsConsoleDriver.cs
 System/__ComObject.cs
 System.Configuration.Assemblies/AssemblyHash.cs
-System.Configuration.Assemblies/AssemblyHashAlgorithm.cs
```
Comment (marek-safar) [inline mcs/class/corlib/corlib.dll.sources]: Please delete no longer used files

**Comment by marek-safar** [comment]
Needs fixing redundant and invalid TypeForwardedFromAttribute(s)

**Comment by akoeplinger** [comment]
Looks good but it needs a corefx bump to get rid of the wrong TypeForwardedFrom attributes.

**Comment by marek-safar** [comment]
@monojenkins commit apidiff and csproj

**Pull request merged**